tests/test_qlsi.py

```diff
@@ -27,9 +30,33 @@ def test_qlsi_phase():
         assert np.allclose(qlsi.wavefront.max(), 8.179288852406586e-08,
                            atol=0, rtol=1e-12)
 
-        assert qlsi.phase.argmax() == 242294
```

Comment:
Why did you remove these two assert statements?

Reply:
It must have happened somehow when I created `test_qlsi_phase_3d`. Good spot. I re-added them now.